packages/components/src/internal/components/navigation/FolderMenu.tsx

```diff
@@ -27,6 +27,10 @@ export const FolderMenu: FC<FolderMenuProps> = memo(props => {
     const { moduleContext, user } = useServerContext();
     const primaryProductId = getPrimaryAppProperties(moduleContext).productId;
 
+    // TODO: the "user" object here is for the current container, so all of the user.isAdmin checks below are incorrect
```

Comment:
Still TODO?

Reply:
Yes, this was an existing issue and I am not sure exactly how to resolve it so I am going to wait until after this PR to address.